jaq-native/src/lib.rs

```diff
@@ -260,6 +269,163 @@ fn now() -> Result<f64, Error> {
 #[cfg(feature = "std")]
 const STD: &[(&str, usize, RunPtr)] = &[("now", 0, |_, _| box_once(now().map(Val::Float)))];
 
+#[cfg(feature = "math")]
+const MATH: &[(&str, usize, RunPtr)] = &[
+    math::math_0_ary!("acos", libm::acos, math::as_float, Val::Float),
+    math::math_0_ary!("acosh", libm::acosh, math::as_float, Val::Float),
+    math::math_0_ary!("asin", libm::asin, math::as_float, Val::Float),
+    math::math_0_ary!("asinh", libm::asinh, math::as_float, Val::Float),
+    math::math_0_ary!("atan", libm::atan, math::as_float, Val::Float),
+    math::math_0_ary!("atanh", libm::atanh, math::as_float, Val::Float),
+    math::math_0_ary!("cbrt", libm::cbrt, math::as_float, Val::Float),
+    math::math_0_ary!("cos", libm::cos, math::as_float, Val::Float),
+    math::math_0_ary!("cosh", libm::cosh, math::as_float, Val::Float),
+    math::math_0_ary!("erf", libm::erf, math::as_float, Val::Float),
+    math::math_0_ary!("erfc", libm::erfc, math::as_float, Val::Float),
+    math::math_0_ary!("exp", libm::exp, math::as_float, Val::Float),
+    math::math_0_ary!("exp10", libm::exp10, math::as_float, Val::Float),
+    math::math_0_ary!("exp2", libm::exp2, math::as_float, Val::Float),
+    math::math_0_ary!("expm1", libm::expm1, math::as_float, Val::Float),
+    math::math_0_ary!("fabs", libm::fabs, math::as_float, Val::Float),
+    math::math_0_ary!("frexp", libm::frexp, math::as_float, |(m, e)| Val::Arr(
+        Rc::new(vec![Val::Float(m), Val::Int(e as isize)])
+    )),
+    math::math_0_ary!("gamma", libm::tgamma, math::as_float, Val::Float),
+    math::math_0_ary!("j0", libm::j0, math::as_float, Val::Float),
+    math::math_0_ary!("j1", libm::j1, math::as_float, Val::Float),
+    math::math_0_ary!("lgamma", libm::lgamma, math::as_float, Val::Float),
+    math::math_0_ary!("log", libm::log, math::as_float, Val::Float),
+    math::math_0_ary!("log10", libm::log10, math::as_float, Val::Float),
+    math::math_0_ary!("log1p", libm::log1p, math::as_float, Val::Float),
+    math::math_0_ary!("log2", libm::log2, math::as_float, Val::Float),
+    math::math_0_ary!("logb", math::logb, math::as_float, Val::Float),
+    math::math_0_ary!("modf", libm::modf, math::as_float, |(f, r)| Val::Arr(
```

Comment:
I would also test `modf`, because its output is quite different from all other functions.